dev/PushNotifications/PushNotificationUtility.cpp

```diff
@@ -4,6 +4,18 @@
 #include "externs.h"
 #include "PushNotificationUtility.h"
 
+wil::unique_cotaskmem_string GetAppUserModelId()
+{
+    wchar_t appId[APPLICATION_USER_MODEL_ID_MAX_LENGTH] = {};
+    UINT32 appIdSize{ ARRAYSIZE(appId) };
+
+    const HRESULT hr = ::GetCurrentApplicationUserModelId(&appIdSize, appId);
+    
+    THROW_HR_IF(hr, hr != HRESULT_FROM_WIN32(ERROR_SUCCESS));
```

Comment:
Can we simplify to this?

```suggestion
    THROW_IF_FAILED(::GetCurrentApplicationUserModelId(&appIdSize, appId));
```